nextcord/ext/commands/bot.py

```diff
@@ -701,7 +701,7 @@ def _load_from_module_spec(
                     raise RuntimeError(f"""
                     Looks like you are attempting to load an asynchronous setup function incorrectly.
                     Please read our FAQ here:
-                    https://nextcord.readthedocs.io/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it
+                    https://docs.nextcord.dev/en/latest/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it
```

Comment:
```suggestion
                    https://docs.nextcord.dev/en/stable/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it
```

Reply:
the stable branch/link is a bit outdated though